bim_webviewer/bim_web_viewer.md

```diff
@@ -4099,6 +4107,62 @@ There were no actual API changes that necesitated a breaking change here but we
 
 I have the privilege of writing this migration guide from the future and can tell you that we've been able to make it even better without (as much) of a framerate hit for larger models in v6.0.1 and you should consider upgrading to that or later. v3 to v4 may also not have needed a breaking change in retrospect so you can safely go from v3 to v4 without your code breaking but know that rendering will behave and perform differently and hopefully mostly for the better on v4 (but again vastly better on v6).
 
+### Migrating to World Coordinates
+
+<p class="heading-link-container">
+  <a class="heading-link" href="#migrating-to-world-coordinates"></a>
+</p>
+
+#### Context
+
+Historically, many methods and events returned and/or expected to receive coordinates that were not consistent with the model coordinates from the source file. To get the correct coordinates you would need to add/subtract the result of `model.getGlobalOffset` to them. This would affect models that are significantly offset from the origin, which we refer to as being in "world coordinates".
+
+We have since taken the stance that all coordinates returned or required as parameters to the Web Viewer should be consistent with the model coordinates from the source file. However, we will be updating them as we come across them so they may be shipped across multiple breaking changes.
+
+#### Migration Guide
+
+If you were saving data returned from a method or event payload that has changed, that data may now be inconsistent if there is a global offset (i.e. if `model.getGlobalOffset` is a non-zero vector) for that model. This can result in behavior where setting the camera position with `setPosition` may be very far away from the actual model. To migrate the old data you would need to translate by the `model.getGlobalOffset` to be in the correct coordinate system.
+
+For example, say you were on an old version of the Web Viewer in which `camera.getPosition/setPosition` did not operate on "world coordinates" and you were saving the camera positions with the following:
+
+```ts
+const cameraPosition = await viewer.camera.getPosition();
+
+postCameraPositionToServer(cameraPosition);
+```
+
+And then you wanted to load these saved positions to set the initial camera position:
+
+```ts
+const { x, y, z } = await getCameraPositionFromServer();
+
+viewer.camera.setPosition(x, y, z);
+```
+
+But then a new version of the Web Viewer is released that updates `camera.getPosition/setPosition` to be in "world coordinates". The values you have saved in your DB will now be incorrect when you pass them to `setPosition`.
```

Comment:
Do we maybe want to mention which version of the SDK introduced this change? Feels ambiguous otherwise hehe.

Reply:
This is just an example of the type of problem you can run into with these changes. Each of the specific changes are in their respective migration guides. This one about `get/setPosition` is in the v7 to v8 section. 

Was it not clear that this was an abstract example? For that perhaps I could change the section title here from "Migration Guide" to "Migration Guide Example"?

---

Or given that it was clear, do you still think it should reference this as a concrete example? Perhaps I could say something like:
```md
For example, say you were on an old version of the Web Viewer in which camera.getPosition/setPosition did not operate on "world coordinates" (e.g. v6) and you were saving the camera positions with the following:

/* ... and then later...  */

But then a new version of the Web Viewer is released that updates `camera.getPosition/setPosition` to be in "world coordinates" (e.g. v7). The values you have saved in your DB will now be incorrect when you pass them to `setPosition`.
```

I think it makes this less clear as an abstract example if we reference specific versions here, but it's definitely subjective.

Reply:
If I follow a link to this section, I think thats fine.

But if I read this in isolation, I feel like I am missing some context and feel unsure if this guide is for me or not. 

Not sure how often that would happen, it is at the end and it is linked multiple times. But I don't think it could hurt to add a bit there. Even if its

```
Historically (prior to v8), many methods and events...
```

Reply:
Ah, actually I see. You don't want to tie it to a version because we might later provide a method that is in local coordinates (maybe by mistake hehe).

Its all good, this section is good to go.